languagetool-core/src/main/resources/org/languagetool/resource/spelling_global.txt

```diff
@@ -19997,4 +19997,268 @@ DB Schenker
 Schenker AG
 Arda Güler
 Prime Reading
-Amazon Freevee
\ No newline at end of file
+Amazon Freevee
+Anna Márton
+Carol Levy
+Elizabeth Hussey
+Jane Grey
+Jane Leeves
+Jane Moraes
+Joan Gregson
+Joan Marsh
+Joan Martí i Alanis
+Joan Riviere
+Julia Görges
+Maria Koller
+Maria Melita
+Paula Grace
+Paula Borges
+Rose Parish
+Beni Johnson
+Jennifer Walton
+Marguerite Durand
+María Moliner
+Renee Walker
+Alexis Rose
+Josie Maran
+Leonora Fani
+Marnie Simpson
+María Gabriela
+Adrian Fernández
+Alberto Setta
+Alberto Youssef
+Alessandro Safina
+Alfredo Capelli
+Amos Tucker
+Andrew Jacobson
+Andrew Marton
+André Masséna
+Anthony Martial
+Antoine Fabre d'Olivet
+Antonio Cesaro
+Antonio Maisto
+Arthur Harper
+Ben Cardin
+Ben Johnson
+Bill Hayden
+Bob Doyle
+Brian Branston
+Brian Wilcox
+Carlo Briani
+Carlos Fairbanks
+Carlos García
+Carlos Mardel
+Carlos María de Castro
+Carlos Mayolo
+Central Perk
+Charles Barlet
+Charles Gardner
+Charles Marker
+Charles Moulton
+Charles Napier
+Charles Packer
+Chris Colfer
+Chris Connelly
+Chris Conner
+David Harvey
+David Willis
+Eduardo Martini
+Edward Earl
+Edward Pescott
+Elliot Carver
+Francesco Cavallini
+Francisco Galdós
+Franz Hössler
+François Sagat
+Frédéric Sauser
+Gary Goldman
+George Busk
+George Harris
+George Saunders
+George Sisler
+Giovanni della Robbia
+Jack Kirby
+Jaime Murray
+James Bagwell
+James Boyd
+James Maslow
+Jan Schäfer
+Jean Boivin
+Jean Charcot
+Jean Martinon
+Jim Calder
+Jim Farber
+John Ball
+John Barlow
+John Berry
+John Brahm
+John Browning
+John Burton
+John Cochrane
+John Decker
+John Dee
+John Dodson
+John Dutton
+John Fassel
+John Graham
+John Hart
+John Kao
+John Kerr
+John Landon
+John Leckie
+John Leland
+John Mallas
+John Marston
+John Martyn
+John Michaels
+John Mitchell
+John Molson
+John Nicks
+John Payne
+John Pilger
+John Price
+John Sloman
+John Teller
+John Thurber
+John Walton
+John Wisker
+Jonas Falk
+José Marzán
+Joseph Bové
+Joseph Cilona
+Josué Montello
+José Blanc de Portugal
+José Mouzinho d'Albuquerque
+José Mouzinho de Albuquerque
+Juan Lavalle
+Juan Martín del Potro
+Julián Marías
+Júnior Díaz
+Kevin Johnson
+Leo Ferris
+Lucas Fernández
+Luís Thomar
+Lê Duẩn
+Manuel Murguía
+Marcel Dussault
+Marcelo Esberard
+Marcus Harvey
+Mario Zanini
+Mark Heller
+Mark Millar
+Mark Robson
+Mark Spitzer
+Mark Waid
+Martin Sostre
+Max Müller
+Michael Barber
+Michael Bolotin
+Michael Carlin
+Michael Harner
+Michael Morton
+Michael Norman
+Michael Steger
+Michael Tejera
+Michael Warner
+Miguel Burnier
+Mike Penner
+Mikhail Yuryevich
```

Comment:
Avoid Russian names in global_spelling.txt because different languages use different spellings. 
Is this one? https://pt.wikipedia.org/wiki/Mikhail_L%C3%A9rmontov